package.json

```diff
@@ -2,28 +2,25 @@
   "name": "splunk-events",
   "version": "1.3.4",
   "description": "Javascript lib to create Splunk Logs via HTTP",
-  "main": "SplunkEvents.js",
+  "main": "lib/splunk-events.js",
+  "types": "lib/splunk-events.d.ts",
   "scripts": {
     "test": "vtex-test-tools test",
-    "build": "babel src/index.js -o SplunkEvents.js && yarn test",
+    "build": "tsc -D --outDir lib ./src/splunk-events.ts",
+    "lint": "eslint --ext ts .",
     "prereleasy": "yarn test",
     "postreleasy": "npm publish",
-    "prepublish": "yarn test && yarn build"
+    "prepublishOnly": "yarn test && yarn build"
```

Comment:
Why the only?

Reply:
so it doesn't run on `yarn install`, only before the `yarn publish`